back-end/photo_app/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib import admin
+from django.urls import path, include
+from rest_framework import routers
+from .views import PhotoViewSet
+
+router = routers.DefaultRouter()
+router.register(r'user', PhotoViewSet)
+
+urlpatterns = [
+  path('', include(router.urls)),
+  path('admin/', admin.site.urls)
```

Comment:
The `admin/` route should be defined in the project-level `urls.py`, not within the `photo_app` app.